src/lib.rs

```diff
@@ -14,7 +14,7 @@ impl Format {
     }
 }
 
-/// Describes a source image which can be used for with `downsample`
+/// Describes a source image which can be used for `downsample`
```

Comment:
Let's also make this an intradoc link then:

```suggestion
/// Describes a source image which can be used for [`downsample()`]
```